### DISCUSSION TITLE
Clear the cache when making a new deployment.

### DISCUSSION THREAD

**Discussion by EliseiNicolae** [vercel/community#2358] [Help] [answered]
### Summary

Hi folks!

I have a subdomain and the production URL stored in Vercel.
I want when I'm making a new deployment to clear the cache of the old build. How can I do that?

Here's my CLI commands to make a new deploy to staging:

```
	vercel pull --yes --environment=preview
	vercel build
	vercel deploy --prebuilt
```

vercel alias URL_GENERATED_ABOVE staging.test.com

**Comment by swarnava** [comment, marked as answer]
Hi, you can pass `--force` param while deploying your app to skip build cache.

**Reply by EliseiNicolae** [reply]
Hi @swarnava, thanks for your response.
Can you confirm if it will still cache the data after using --force?

**Reply by swarnava** [reply]
`--force` will skip build cache, your application cache will still work as configured

**Reply by EliseiNicolae** [reply]
Ok, thank you @swarnava.
